### PR TITLE
MTM-44133 related topics for device management

### PR DESCRIPTION
**Pull request by rssag**
i didn't find anything suitable for simulators.

### CODE REVIEW

File: content/users-guide/device-management-bundle/connecting-devices.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ helpcontent:
     **Accepted** - the user has allowed the credentials to be send to the device"
 ---
 
+{{< c8y-admon-related >}}
+* See [Device integration tutorials](/device-tutorials/tutorials-introduction) for information on the integration of specific devices
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/connecting-devices.md:22]: @rssag @MWindel We need to decide on various format questions and unify accordingly. 
I always use a full stop at the end.
I do not use "See" at the beginning.

File: content/users-guide/device-management-bundle/connectivity.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ aliases:
   - users-guide/optional-services/#connectivity
 ---
 
+{{< c8y-admon-related >}}
+* See [Reference guide > Device management library > Connectivity](/reference/device-management-library/#connectivity) for the device management library entry on connectivity
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/connectivity.md:10]: Our format is Device management library > Connectivity in the link, followed by "in the Reference guide"

I would at least also link to the connectivity settings: https://cumulocity.com/guides/users-guide/administration/#connectivity

File: content/users-guide/device-management-bundle/connecting-devices.md [hunk 3]
```diff
@@ -18,6 +18,12 @@ helpcontent:
...
+* See the [Device SDK guide](/device-sdk/introduction/) for information on device integration via REST and C++
+* See the [device control API](https://cumulocity.com/api/{{< c8y-current-version >}}/#tag/Device-control-API) for REST API methods concerning device management
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/connecting-devices.md:24]: Perhaps add one more: See the documentation for the protocols in the Protocol integration guide for registering devices of a specific type?

File: content/users-guide/device-management-bundle/device-details.md [hunk 4]
```diff
@@ -3,6 +3,12 @@ weight: 31
 title: Device details
 layout: redirect
 ---
+
+{{< c8y-admon-related >}}
+* See [Reference guide > Device management library > Connectivity](/reference/device-management-library/#device-details) for the device management library entry on device details
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/device-details.md:8]: see above

File: content/users-guide/device-management-bundle/connecting-devices.md (hunk 1 above)
Comment (rssag) [reply content/users-guide/device-management-bundle/connecting-devices.md:22]: added full stops, removed "see", and done @simple demo devices.

File: content/users-guide/device-management-bundle/device-details.md (hunk 4 above)
Comment (rssag) [reply content/users-guide/device-management-bundle/device-details.md:8]: done

File: content/users-guide/device-management-bundle/connectivity.md (hunk 2 above)
Comment (rssag) [reply content/users-guide/device-management-bundle/connectivity.md:10]: fixed format.

File: content/users-guide/device-management-bundle/connecting-devices.md (hunk 3 above)
Comment (rssag) [reply content/users-guide/device-management-bundle/connecting-devices.md:24]: added the entry

File: content/users-guide/device-management-bundle/connecting-devices.md [hunk 5]
```diff
@@ -18,6 +18,13 @@ helpcontent:
     **Accepted** - the user has allowed the credentials to be send to the device"
 ---
 
+{{< c8y-admon-related >}}
+* [Device integration tutorials](/device-tutorials/tutorials-introduction) for information on the integration of simple demo devices.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/connecting-devices.md:22]:
```suggestion
* [Device integration tutorials](/device-tutorials/tutorials-introduction) for step-by step instructions on registering simple demo devices.
```

File: content/users-guide/device-management-bundle/connecting-devices.md [hunk 6]
```diff
@@ -18,6 +18,13 @@ helpcontent:
...
+* The [Device SDK guide](/device-sdk/introduction/) for information on device integration via REST and C++.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/connecting-devices.md:23]:
```suggestion
* The [Device SDK guide](/device-sdk/introduction/) for detailed descriptions on device integration via REST and C++.
```

File: content/users-guide/device-management-bundle/connecting-devices.md [hunk 7]
```diff
@@ -18,6 +18,13 @@ helpcontent:
...
+* The [device control API](https://cumulocity.com/api/{{< c8y-current-version >}}/#tag/Device-control-API) for REST API methods concerning device management.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/connecting-devices.md:24]: concerning device management > is this only one API for the entire device management?

File: content/users-guide/device-management-bundle/connectivity.md [hunk 8]
```diff
@@ -6,6 +6,10 @@ aliases:
   - users-guide/optional-services/#connectivity
 ---
 
+{{< c8y-admon-related >}}
+* [Device management library > Connectivity](/reference/device-management-library/#connectivity) in the *Reference guide* for the device management library entry on connectivity.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/connectivity.md:10]: Didn´t I add a comment somewhere that a link to the connectivity settings is missing? Can´t find it :-).
https://cumulocity.com/guides/users-guide/administration/#connectivity

File: content/users-guide/device-management-bundle/managing-device-types.md [hunk 9]
```diff
@@ -11,6 +11,12 @@ helpcontent:
   To add a new device protocol, click **Add device protocol** at the top right, select a device protocol type and configure it following the instructions for the particular type in the *Protocol integration guide*."
 ---
 
+{{< c8y-admon-related >}}
+* [Interfacing devices](/concepts/interfacing-devices/) in the *Concepts guide* for more information on how to interface devices.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/managing-device-types.md:15]:
```suggestion
* [Interfacing devices](/concepts/interfacing-devices/) in the *Concepts guide* for information on the concepts relevant for interfacing IoT devices and other IoT-related data sources with {{< product-c8y-iot >}}.
```

File: content/users-guide/device-management-bundle/managing-device-types.md [hunk 10]
```diff
@@ -11,6 +11,12 @@ helpcontent:
...
+* [LPWAN Custom device protocols](/protocol-integration/lpwan-custom-codec/) in the *Protocol integration guide* for custom device protocols for LPWAN devices.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/managing-device-types.md:16]:
```suggestion
* [LPWAN Custom device protocols](/protocol-integration/lpwan-custom-codec/) in the *Protocol integration guide* for details on custom device protocols for LPWAN devices.
```

File: content/users-guide/device-management-bundle/smartrest-templates.md [hunk 11]
```diff
@@ -14,6 +14,11 @@ helpcontent:
     To create a new template, click **Create template** at the top right, provide a name and unique ID and add the required messages or responses as described in the *User guide*."
 ---
 
+{{< c8y-admon-related >}}
+* [SmartREST 2.0](/reference/smartrest-two/) in the *Reference guide* for more information on SmartREST.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/smartrest-templates.md:18]:
```suggestion
* [SmartREST 2.0](/reference/smartrest-two/) in the *Reference guide* for details on the SmartREST protocol, the data format used, as well as the anatomy and registration of SmartREST templates.
```

File: content/users-guide/device-management-bundle/connecting-devices.md (hunk 7 above)
Comment (rssag) [reply content/users-guide/device-management-bundle/connecting-devices.md:24]: no, but "device management" in this place is too generic.
i specified the path to "New device request API" and changed the text accordingly.

**Comment by rssag** [comment]
for device details, there are no related topics anymore.
i feel like there are a lot of links already in the text. we could add more, for all the subsections, but that would probably result in a very bloated related topics section. what do you think?

File: content/users-guide/device-management-bundle/connectivity.md [hunk 12]
```diff
@@ -6,6 +6,11 @@ aliases:
   - users-guide/optional-services/#connectivity
 ---
 
+{{< c8y-admon-related >}}
+* [Device management library > Connectivity](/reference/device-management-library/#connectivity) in the *Reference guide* for the device management library entry on the `c8y_Mobile` fragment used in managed objects.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/connectivity.md:10]:
```suggestion
* [Device management library > Connectivity](/reference/device-management-library/#connectivity) in the *Reference guide* for details on the `c8y_Mobile` fragment used in managed objects.
```

Comment (BeateRixen) [reply content/users-guide/device-management-bundle/connectivity.md:10]: How about that?

File: content/users-guide/device-management-bundle/connectivity.md [hunk 13]
```diff
@@ -6,6 +6,11 @@ aliases:
...
+* [Administration > Changing settings > Managing the connectivity settings](/users-guide/administration/#connectivity) in the *User guide* for how to change the connectivity settings via the UI.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/connectivity.md:11]:
```suggestion
* [Administration > Changing settings > Managing the connectivity settings](/users-guide/administration/#connectivity) in the *User guide* for information on how to change the connectivity settings via the UI.
```

File: content/users-guide/device-management-bundle/connecting-devices.md [hunk 14]
```diff
@@ -18,6 +18,13 @@ helpcontent:
     **Accepted** - the user has allowed the credentials to be send to the device"
 ---
 
+{{< c8y-admon-related >}}
+* [Device integration tutorials](/device-tutorials/tutorials-introduction) for step-by step instructions on registering simple demo devices.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/connecting-devices.md:22]:
```suggestion
* [Device integration tutorials](/device-tutorials/tutorials-introduction) for step-by-step instructions on registering simple demo devices.
```

File: content/users-guide/device-management-bundle/device-details.md [hunk 15]
```diff
@@ -3,6 +3,11 @@ weight: 31
 title: Device details
 layout: redirect
 ---
+
+{{< c8y-admon-related >}}
+Refer to the hyperlinks throughout this chapter for related topics.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/device-details.md:8]:
```suggestion
References to related content in the documentation are mentioned in the details descriptions below.
```

**Review by BeateRixen — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Comment by BeateRixen** [comment]
@Oezge-Akin Hi Özge, we would like to add some "related topics" at the top of each chapter to guide the users to other places of interest in the documentation. We have already added what we found useful but perhaps you have some more/better ideas? Could you please review and comment if you think we should add links? Thanks!

File: content/users-guide/device-management-bundle/connecting-devices.md [hunk 16]
```diff
@@ -33,6 +33,13 @@ helpcontent:
     To register a device, click **Register device** at the right of the top bar, select an option from the dropdown list and follow the instructions in the device registration wizard."
 ---
 
+{{< c8y-admon-related >}}
+* [Device integration > Integration tutorials](/device-integration/integration-tutorials) for step-by-step instructions on registering devices.
+* [Device integration using REST](/device-integration/rest/) for a detailed descriptions on device integration via REST.
+* The [New device requests API](https://cumulocity.com/api/core/{{< c8y-current-version >}}/#tag/New-device-requests) for REST API methods concerning the creation of new devices.
```
Comment (MWindel) [inline content/users-guide/device-management-bundle/connecting-devices.md:39]:
```suggestion
* The [New device requests API](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/New-device-requests) for REST API methods concerning the creation of new devices.
```

File: content/users-guide/device-management-bundle/connecting-devices.md [hunk 17]
```diff
@@ -33,6 +33,13 @@ helpcontent:
...
+* The [Protocol integration guide](/protocol-integration/overview) for details on registering devices using various standard protocol types.
```
[the remark this answers is not in the document]
Comment (rssag) [reply content/users-guide/device-management-bundle/connecting-devices.md]: made it the first entry

File: content/users-guide/device-management-bundle/grouping-devices.md [hunk 18]
```diff
@@ -13,6 +13,14 @@ helpcontent:
     You can easily restructure groups or assign devices to groups by dragging and dropping groups or devices in the navigator."
 ---
 
+{{< c8y-admon-related >}}
+* [Cockpit > Managing assets > Assets hierarchy](/users-guide/cockpit/#managing-assets) in the *User guide* for information on the asset hierarchy, assets and groups.
+* [Administration > Managing permissions > Inventory roles](/users-guide/administration/#inventory-roles) in the *User guide* on how to assign inventory roles to groups of devices.
+* The [managed objects API](https://cumulocity.com/api/core/{{< c8y-current-version >}}/#tag/Managed-objects) for REST API methods concerning managed objects (devices or groups of devices).
+* The [operations API](https://cumulocity.com/api/core/{{< c8y-current-version >}}/#tag/Operations) for REST API methods concerning operations.
```
[the remark this answers is not in the document]
Comment (rssag) [reply content/users-guide/device-management-bundle/grouping-devices.md]: removed the entry

File: content/users-guide/device-management-bundle/device-details.md [hunk 19]
```diff
@@ -3,6 +3,11 @@ weight: 31
 title: Device details
 layout: redirect
 ---
+
+{{< c8y-admon-related >}}
+References to related content in the documentation are mentioned in the details descriptions below.
```
[the remark this answers is not in the document]
Comment (rssag) [reply content/users-guide/device-management-bundle/device-details.md:8]: there's nothing in this related admonition, but it's a pointer to the subsections below.

File: content/users-guide/device-management-bundle/managing-device-types.md [hunk 20]
```diff
@@ -11,6 +11,12 @@ helpcontent:
   To add a new device protocol, click **Add device protocol** at the top right, select a device protocol type and configure it following the instructions for the particular type in the *Protocol integration guide*."
 ---
 
+{{< c8y-admon-related >}}
+* [Interfacing devices](/concepts/interfacing-devices/) in the *Concepts guide* for information on the concepts relevant for interfacing IoT devices and other IoT-related data sources with {{< product-c8y-iot >}}.
+* [LPWAN Custom device protocols](/protocol-integration/lpwan-custom-codec/) in the *Protocol integration guide* for details on custom device protocols for LPWAN devices.
```
[the remark this answers is not in the document]
Comment (rssag) [reply content/users-guide/device-management-bundle/managing-device-types.md]: generalized the entry

**Review by Oezge-Akin — APPROVED** [review]

**Comment by rssag** [comment]
cherry-picked manually to 10.18.0.

**Pull request merged**